src/shadow-embed.js

```diff
@@ -692,3 +696,20 @@ export class ShadowDomWriterBulk {
     this.onEnd_();
   }
 }
+
+/*
+ * Remove any noscript elements.
+ * @param {!Element} parent
+ *
+ * According to the spec (https://w3c.github.io/DOM-Parsing/#the-domparser-interface),
+ * with `DOMParser().parseFromString`, contents of `noscript` get parsed as markup,
+ * so we need to remove them manually.
+ * Why? ¯\_(ツ)_/¯
+ * `createHTMLDocument()` seems to behave the same way.
+ */
+function removeNoScriptElements(parent) {
+  const noscriptElements = toArray(childElementsByTag(parent, 'noscript'));
```

Comment:
Is there a particular reason you use `toArray` and `forEach` over `iterateCursor`?

Reply:
didn't know about `iterateCursor`, neat!